lib/sdr_client/cli.rb

```diff
@@ -38,7 +38,8 @@ def self.start(command, options)
       case command
       when 'deposit', 'register'
         display_errors(validate_deposit_options(options))
-        SdrClient::Deposit.run(accession: command == 'deposit', **options)
+        job_id = SdrClient::Deposit.run(accession: command == 'deposit', **options)
+        poll_for_job_complete(job_id: job_id, url: options[:url]) # TODO: add an option that skips this
```

Comment:
Let the user know that it is waiting? Wait more than 25 secs?

Reply:
What I've found so far is that if this time period elapses, there's an problem upstream.

Reply:
I'm certainly up for changing this later if we see a need for that.